gnuradio-runtime/lib/terminate_handler.h

```diff
@@ -0,0 +1,19 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2018 Free Software Foundation, Inc.
+ *
+ * This file is part of GNU Radio
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+#ifndef INCLUDED_GR_TERMINATE_HANDLER_H
+#define INCLUDED_GR_TERMINATE_HANDLER_H
+
+#include <gnuradio/api.h>
+
+namespace gr {
+void install_terminate_handler() GR_RUNTIME_API;
```

Comment:
Again some backward GCC-stuff. This doesn't create a valid `dllexport` directive. This does:
```diff
--- a/terminate_handler.h 2020-12-19 16:41:48
+++ b/terminate_handler.h 2020-12-19 17:19:38
@@ -13,7 +13,7 @@
 #include <gnuradio/api.h>

 namespace gr {
-void install_terminate_handler() GR_RUNTIME_API;
+void GR_RUNTIME_API install_terminate_handler();
 }

 #endif /* INCLUDED_TERMINATE_HANDLER_H */
```